sig-scheduling/CONTRIBUTING.md

```diff
@@ -72,6 +72,26 @@ to the issue/PR, which helps to preserve the context.
 
 * Always open an issue for a TODO or a follow-up just in case you forget it.
 
+### Technical and style guidelines
```

Comment:
Maybe also adds links to other sig's guidelines. The ones are essential to sig-scheduling I can think of includes:

- API convention - https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
- Contextual/structrural Logging convention - [contributors/devel/sig-instrumentation/logging.md](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)
- ...

Reply:
> Contextual/structrural Logging convention - [contributors/devel/sig-instrumentation/logging.md](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)

Mentioning log level also would be valuable. Anyway, referring to the logging docs is valuable.